tsconfig.json

```diff
@@ -12,7 +12,14 @@
     "esModuleInterop": true,
     "baseUrl": "./packages",
     "paths": {
-      "@ash.ts/*": ["./*"]
```

Comment:
I think that changing this to `"@ash.ts/*": ["./*/src"]` should be enough. 

One thing that is broken in both cases is the documentation starting page. It should be like on: https://icek.github.io/ash/ but it's `signals` package README now. To fix that we should add: `"readme": "packages/ash/README.md",` in `typedocOptions` sections of this file.